libs/core/langchain_core/load/mapping.py

```diff
@@ -126,12 +126,12 @@
         "agents",
         "AgentActionMessageLog",
     ),
-    ("langchain", "schema", "agent", "OpenAIToolAgentAction"): (
+    ("langchain", "schema", "agent", "ToolAgentAction"): (
         "langchain",
         "agents",
         "output_parsers",
-        "openai_tools",
-        "OpenAIToolAgentAction",
+        "tools",
+        "ToolAgentAction",
```

Comment:
Don't think it matters too much in this particular case since actions aren't really serialized but still technically breaking

Reply:
yea probably shouldn't change namespace

Reply:
or should support both

Reply:
updated